Sources_v3/Controllers/ChannelListController/ChannelListController_Tests.swift

```diff
@@ -64,14 +64,25 @@ class ChannelListController_Tests: StressTestCase {
         XCTAssertEqual(controller.state, .remoteDataFetched)
     }
     
-    func test_startUpdating_changesControllerStateOnError() {
+    func test_channelsAccess_changesControllerState() {
```

Comment:
Hmm, accessing channels should do nothing to the state, no?

Reply:
Discussed in Slack.